Makefile

```
@@ -0,0 +1,6 @@
+ifneq (,$(wildcard ./.env))
+	include .env
+endif
+
+interfaces: 
```

Rename this make rule by `generate-interfaces` for more clarity.
```suggestion
generate-interfaces: 
```